lighthouse-cli/test/fixtures/delayed-fcp.html

```diff
@@ -0,0 +1,12 @@
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
+<body>
+  <script>
+    setTimeout(() => {
```

Comment:
maybe add a comment as to the goal of this page?

Reply:
done